src/com/svyrydova/Modul1/Task1/UserInputService.java

```diff
@@ -0,0 +1,48 @@
+package com.svyrydova.Modul1.Task1;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class UserInputService {
+    static Scanner scanner = new Scanner(System.in);
+    static int input = scanner.nextInt();
+
+    public static int getInput() {
+        return input;
+    }
+
+    public static void UserInputService(Array array){
```

Comment:
A method should not be named the same as a class unless it is a constructor. And the name with lowercase letters